### DISCUSSION TITLE
Please make it possible to subscribe to failed workflows on the default branch independent of the triggering user

### DISCUSSION THREAD

**Discussion by peterjuras** [community/community#55379] [Actions]
### Select Topic Area

Product Feedback

### Body

Hi,

Currently it's only possible to receive failed notifications of GitHub actions workflows if I trigger them (be it via push or a scheduled workflow that I created updated).

I'd like to see all failed workflows (at least for default branches) in my GitHub notifications, regardless of who broke the build. We have a lot of automatic merges from bots (dependabot, renovate, etc.) and these will be missed without these notifications.

As a workaround, it is possible to add a step that sends an email, but this is cumbersome to do for all repositories and is not the same as seeing it within the GitHub notifications section.

Please consider adding this functionailty.

Thanks,
Peter

**Comment by lmjhs** [comment]
I completely agree, this would be such a valuable feature.  We currently have to use either a `failed()` job at the end of every workflow or use the webhook functionality.  It would be so much neater to allow notifications to be sent to a user / team from either the settings or the workflow definition.

I propose the following changes:

1. Set a default failure notification list of users / teams for all workflows in the repository settings (maybe in Actions > General).
2. Add an additional configuration property to the workflow definition to be able to override the default to specify a list of users / teams that should receive failure notifications for that workflow.

**Reply by AHJiapengZhang** [reply]
+1. It is such a design flaw that only one person can receive failure notification. It's simply ridiculous. No one be there in a team for all the time to be alerted for the failed events. There should be the flexibility within GitHub to specify who receive what.

**Reply by duaneking** [reply]
I want to call out the fact that not even people at Github are getting these messages because these kind of things are being assigned to people that don't even exist at Github anymore - https://github.com/integrations/slack/issues/1563 is the perfect example with @ashokirla assigned a task that has not been finished for years because they probably don't even work there anymore - and we don't have as a community a way to escalate to their manager or GitHub, because there's no way to do that on Github, because GitHub leadership has chosen not to value its employees enough to think about succession for projects that they work on.

**Comment by AHJiapengZhang** [comment]
It might help if anyone can tag relevant people in the github action team.

**Reply by duaneking** [reply]
There should be a GitHub Action Team user or something so that when a task is assigned to a GitHub employee and they don't get it done for years (because they left the company or GitHub otherwise had retention issues), the community can escalate.
